classroom.rb

```diff
@@ -0,0 +1,9 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
```

Comment:
- [ ] there should be a method that serves as the 'has-many' side (a classroom has many students). have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) with examples. this might look something like below:
```
  def add_student(student)
    @students << student
    student.classroom = self
  end
```